lib/utils/HttpHelper.dart

```diff
@@ -99,7 +94,6 @@ class HttpHelper {
     if (isTestModeActive) {
       var token = 'Basic ' + base64Encode(utf8.encode('off:off'));
       headers.addAll({'authorization': token});
-      print("TEST-MODE");
```

Comment:
Maybe we can keep this log to confirm to the user that he is on test mode. 
it is not that big deal, and this is not a blocker to merge.

Reply:
Hi @MohamedFBoussaid thanks for taking a look. If this print statement is there it is printed with every test, see
![image](https://user-images.githubusercontent.com/5388919/107055416-9163e180-67d1-11eb-9fad-c0a69e38a97c.png)
Would you be OK with leaving it out?
If it is not there the output looks more pleasant
![image](https://user-images.githubusercontent.com/5388919/107055677-d38d2300-67d1-11eb-9415-665946c63418.png)

If you still prefer to keep it there then I can add the statement back in.
What do you think?

Reply:
In that case let's just delete it, it does not worth it.

Reply:
Will do, thanks!